dev-packages/application-package/src/application-props.ts

```diff
@@ -63,7 +63,7 @@ export namespace FrontendApplicationConfig {
     export const DEFAULT: FrontendApplicationConfig = {
         applicationName: 'Eclipse Theia',
         defaultTheme: 'dark',
-        defaultIconTheme: 'none',
+        defaultIconTheme: 'theia-file-icons',
```

Comment:
I believe we should probably add a breaking changes entry for it (breaking behavior). 
Applications which expected `none` will now get `theia-file-icons`.

Reply:
Yeah, definitely makes sense 👍